integration_tests/pages/bodyscan/bodyScanConfirmation.ts

```diff
@@ -2,7 +2,7 @@ import Page, { PageElement } from '../page'
 
 export default class BodyScanConfirmation extends Page {
   constructor() {
-    super('Body scan recorded', { hasBackLink: false })
+    super('Body scan recorded', { hasBackLink: false }, { hasFeedbackBanner: true })
   }
```

Comment:
If we're importing `Page` is `hasFeedbackBanner: true` not the default? Not sure this is needed here but could be wrong.